src/utils/Logger.h

```diff
@@ -124,12 +125,11 @@ class Logger {
 public:
 	/** Initializes the log level, log stream and the list of log level names.
 	 * If ENABLE_MPI is enabled by default all process perform logging output. */
-	Logger(logLevel level = Log::Error, std::ostream *os = &(std::cout));
+	Logger(logLevel level = Log::Error, std::ostream *os = &(std::cout));  // Write to stream
 
-	Logger(logLevel level, std::string prefix);
+	Logger(logLevel level, std::string prefix);  // Write to file
```

Comment:
```suggestion
	/**
	 * Constructor for a logger to a stream.
	 *
	 * Initializes the log level, log stream and the list of log level names.
	 * If ENABLE_MPI is enabled by default, all process perform logging output.
	 */
	Logger(logLevel level = Log::Error, std::ostream *os = &(std::court));

	/**
	 * Constructor for a logger to a file.
	 *
	 * Initializes the log level, log stream and the list of log level names.
	 * If ENABLE_MPI is enabled by default, all process perform logging output.
	 */
	Logger(logLevel level, std::string prefix);
```
More consistent documentation.